src/rpc/server/scala/serv/src/main/scala/com/ctc/omega_edit/grpc/EditorService.scala

```diff
@@ -491,8 +491,10 @@ class EditorService(implicit val system: ActorSystem) extends Editor {
       .transform {
         case Success(_) =>
           Success(ServerControlResponse(kind, getServerPID(), 0))
-        case Failure(_) =>
+        case Failure(e) => {
+          (editors ? LogOp("debug", e))
```

Comment:
What is the advantage of adding a `logOp` over just calling `log.debug`?

Reply:
So from `EditorService` you can not call `log.debug`. You can from `Editors` because it has `with ActorLogging`. So the `LogOp` was my way of making a central location for all logs. The advantage is you can just more than just `log.debug`. With `LogOp` you can do `debug`, `info`, `error` and `warning`.

Reply:
Okay, plus we can later configure them more using say application.conf like akka does for for enabling/disabling say debug logs entirely, not even reaching logback filters.